docs/design/autoware-interfaces/ad-api/features/.pages

```diff
@@ -0,0 +1,2 @@
+nav:
+  - Features: index.md
```

Comment:
You can use `...` to add the rest items.
https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin#rest

```suggestion
  - Features: index.md
  - ...
```

Reply:
Should the rest items be displayed in the menu? I think there will be more than 10 feature pages. If there are too many items in the menu, it will be difficult to find the page. So the rest of the pages will be linked from the index.md.

Reply:
At least for now (since there are only two items), I personally feel adding rest items is better.
But I'll leave it to you.

Reply:
I'll resolve this for now.
If we split out `AD API` pages from the `Design` and create a dedicated section, it might be better to expand each API page.
Let's consider this in the future.